modules/core/src/main/java/org/apache/ignite/internal/processors/cache/transactions/IgniteInternalTx.java

```diff
@@ -44,7 +45,7 @@
 /**
  * Transaction managed by cache ({@code 'Ex'} stands for external).
  */
-public interface IgniteInternalTx {
+public interface IgniteInternalTx extends TransactionView {
```

Comment:
Please, prefer composition over inheritance. Actually `IgniteInternalTx` is not `TransactionView`. 
Composition is also consistent with other existing  view implentations.

Reply:
Fixed.